src/bin/signal_handler.rs

```diff
@@ -14,21 +14,28 @@
 
 #[cfg(unix)]
 mod imp {
+    use std::thread;
+    use std::time::Duration;
     use std::sync::Arc;
+    use std::sync::atomic::{AtomicBool, Ordering};
 
     use rocksdb::DB;
+    use jemallocator;
+    use libc::c_char;
 
     use tikv::server::Msg;
     use tikv::util::transport::SendCh;
     use prometheus::{self, Encoder, TextEncoder};
 
     const ROCKSDB_DB_STATS_KEY: &'static str = "rocksdb.dbstats";
     const ROCKSDB_CF_STATS_KEY: &'static str = "rocksdb.cfstats";
+    const DUMP_FILE_NAME: *const c_char = 0 as *const c_char;
```

Comment:
It's a null pointer. Using ptr::null() requires an extra feature, so use 0 here.